app/views/support_tickets/thank_you.html.erb

```diff
@@ -23,11 +23,11 @@
     </h2>
 
     <p class="govuk-body">
-      Our support team has received your message. We aim to respond to you by email within 5 working days.
+      Our support team has received your message. We aim to respond to you by email within 3 working days.
       If you have not heard from us within that time, please check your spam or junk folder.
     </p>
     <p class="govuk-body">
-      We’re experiencing high demand at the moment. We know that 5 days is a bit of a long time to ask you to wait, but we appreciate your patience.
+      We’re experiencing high demand at the moment. We know that 3 days is a bit of a long time to ask you to wait, but we appreciate your patience.
```

Comment:
Is this needed now? I know this was added as a result of the huge spike of tickets back in Jan and asking the user to wait a week before getting a response.

Reply:
Removal approved